doc/.sphinx/_templates/layout.html

```diff
@@ -0,0 +1,7 @@
+{% extends "!layout.html" %}
+
+{% block footer %}
+{{ super() }}
+<div class="footer">YEP THIS IS AN EXAMPLE
```

Comment:
Is this supposed to say this?

Reply:
Nope, but it doesn't render. I'll fix it though.